### PR TITLE
Update version and signing as well

### PR DESCRIPTION
**Pull request by yojagad**

### CODE REVIEW

**Comment by ankitkumarr** [comment]
I am assuming you ran `redirectCompare` and `packageConfigParser`? Also if you wouldn't mind squashing the commits. Looks good to me otherwise.

File: build.fsx [hunk 1]
```diff
@@ -187,6 +189,7 @@ Target "GenerateZipToSign" (fun _ ->
         "Ignite.SharpNetSH.dll"
         "NCrontab.dll"
         "Newtonsoft.Json.dll"
+        "protobuf-net.dll"
```
Comment (ahmelsayed) [inline build.fsx]: just curious why protobuf dll is in v1? was that added to the v2 webjobs project?

Comment (yojagad) [reply build.fsx]: Actually, good point. I'm not sure why it was ever installed in v1 to begin with. Let me see if I can find out

Comment (ankitkumarr) [reply build.fsx]: Looks like it comes from https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script/WebJobs.Script.csproj#L231-L233?

Comment (ahmelsayed) [reply build.fsx]: oh, so it has been there for a while and we just missed it. thanks @ankitkumarr  and @yojagad, I was just curious where it was coming from.

**Comment by yojagad** [comment]
Ran packageConfigParser, redirectCompare and squashed the commits in the latest push.

**Pull request merged**